app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_of_cars: list) -> float:
+        total_sum = 0
+        for car in list_of_cars:
+            if car.clean_mark < self.clean_power:
+                total_sum += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+
+        return total_sum
+
+    def calculate_washing_price(self, car: Car) -> float:
+        cost = (car.comfort_class * (self.clean_power - car.clean_mark)
+                * self.average_rating) / self.distance_from_city_center
+        cost = round(cost, 1)
+
+        return cost
+
+    def wash_single_car(self, car: Car) -> Car:
```

Comment:
You need to check if clean_power is greater than clean_mark here because if you run this method separately from others it will not work correctly